views/welcome.ejs

```diff
@@ -35,4 +36,29 @@
     <p>We collect anonymous analytics using Simple Analytics. View live: <a href="#">simpleanalytics.com/learndatabases.dev</a></p>
 
   </body>
+
+  <style>
```

Comment:
I was about to ask why this `style` element is not in the `head` or `body`. But I scrolled up a little further and now I see that we're including `./partials/head`, which already contains a complete `html` element.

Since we have another `html` element in _this_ template, the resulting HTML to contains two separate `html` elements. That partials template contains many elements that normally would be placed in the head, outside of the `html` element. The resulting HTML source is confusing but somehow parsed by modern browsers.

There are a lot of good opportunities to fix and streamline the code here. But this otherwise solid PR shouldn't be held up by a preexisting condition. This `style` element can be addressed later.

Issue #171 contains a record of the view template issues mentioned here.

Reply:
Nice comment. I didn't realize it. I'll fix soon.

Reply:
nice find @danielthalhuber !

Reply:
thank you for filing the issue.